src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+// font & theme
+
+const Header = () => {
+	return (
+		<header>
+			<nav>
+                {/* <div>Here is a nav</div> */}
+            </nav>
+		</header>
+	);
+};
+
+export default Header;
```

Comment:
you might remove the Header component. As per your latest commit, I don't see any implemenatation of the header in the project.